shell/pages/home.vue

```diff
@@ -238,11 +239,129 @@ export default {
   },
 
   methods: {
+    fetchSecondaryResources(opts: FetchSecondaryResourcesOpts) {
```

Comment:
this should pick out specific resources from the page and fetch them

need to consider what is shown on page whilst this info is missing / loading